windows/internal/smoke_test.bat

```diff
@@ -2,10 +2,7 @@ set SRC_DIR=%~dp0
 
 pushd %SRC_DIR%\..
 
-if "%CUDA_VERSION%" == "102" call internal\driver_update.bat
-if "%CUDA_VERSION%" == "110" call internal\driver_update.bat
-if "%CUDA_VERSION%" == "111" call internal\driver_update.bat
-if "%CUDA_VERSION%" == "112" call internal\driver_update.bat
+if DEFINED CUDA_VERSION call internal\driver_update.bat
```

Comment:
Couldn't CUDA_VERSION also be set to cpu?

Reply:
Good call, I'll change this to check for inequality with cpu